index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <link rel="stylesheet" type= "text/css" href="epl-modern.css">
+    <link rel="stylesheet" href="style.css">
+      
+    <!-- fav icon -->
+    <link rel='icon' href='assets/icons/favicon-32x32.png' type='image/x-icon' sizes="32x32" />
+
+    <title>A modern fixture table for epl</title>
+    <!-- Optional JavaScript -->
```

Comment:
no need for this comment, should put a comment above each <script element> for clarity.